fuel-core/src/database.rs

```diff
@@ -149,11 +197,33 @@ impl AsRef<Database> for Database {
     }
 }
 
-/// Construct an in-memory database
+/// Construct an ephemeral database
+/// uses rocksdb when rocksdb features are enabled
+/// uses in-memory when rocksdb features are disabled
```

Comment:
Hmmmmm. Is this what we want? Even if the `rocksdb` feature is enabled, should it use in-memory if `--db-type in-memory` is set from the command line?

Reply:
ah good catch, I need to use `::in_memory()` instead

Reply:
Should be fixed now, but I changed it in the `fuel_core::service::FuelService::new_node` method.

Reply:
Should the comment be updated?

Reply:
No this isn't on the cli args / config side, this is the default constructor for the db. This way we can use one method to test both in-memory and rocksdb without needing to put feature flags in every test.